sstables/mx/reader.hh

```diff
@@ -48,7 +49,8 @@ mutation_reader make_crawling_reader(
         schema_ptr schema,
         reader_permit permit,
         tracing::trace_state_ptr trace_state,
-        read_monitor& monitor);
+        read_monitor& monitor,
+        sstable::integrity_check integrity = sstable::integrity_check::no);
```

Comment:
We try to avoid introducing new default values for parameters, too many times we regret them later. It is better to force all callers to make a conscious decision. There aren't many callers to this `make_crawling_reader()` anyway.